en/flight_controller/README.md

```diff
@@ -40,6 +40,7 @@ The list of boards documented in this library can be found in the sidebar (repro
   * [MindPX](../flight_controller/mindpx.md)
   * [MindRacer](../flight_controller/mindracer.md)
 * [Kakute F7](../flight_controller/kakutef7.md)
+* [Durandal](../flight_controller/durandal.md)
```

Comment:
FYI added to the index page for autopilots. CHOSE not to add it to the getting started docs on autopilots, though we could if you think it is important.